lib/maths/unittest/CBoostedTreeUtilsTest.cc

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#include <core/CContainerPrinter.h>
+#include <core/CDataFrame.h>
+
+#include <maths/CBoostedTree.h>
+#include <maths/CBoostedTreeFactory.h>
+#include <maths/CBoostedTreeImpl.h>
+#include <maths/CBoostedTreeLoss.h>
+#include <maths/CBoostedTreeUtils.h>
+
+#include <test/CRandomNumbers.h>
+
+#include "BoostedTreeTestData.h"
+#include "maths/COrderings.h"
```

Comment:
This include should be moved up.

Reply:
In fact this include was no longer needed so I just removed in [ea93475](https://github.com/elastic/ml-cpp/pull/1820/commits/ea934756759014d742ec625bde938a3d8cb24eb0).